src/main/scheduler/scheduler.h

```diff
@@ -28,9 +28,11 @@
 #define TASK_PERIOD_MS(ms) ((ms) * 1000)
 #define TASK_PERIOD_US(us) (us)
 
-#define TASK_STATS_MOVING_SUM_COUNT 64
+#define TASK_STATS_MOVING_SUM_COUNT     64
 
-#define LOAD_PERCENTAGE_ONE 100
+#define LOAD_PERCENTAGE_ONE             100
+
+#define SCHED_TASK_DEFER_MASK           7   // Scheduler loop count is masked with this and when 0 long running tasks are processed
```

Comment:
I would prefer bit masks in HEX. I know it is the same in that case.
Maybe we can also align in to the right not to the left. Just cosmetic :)

Reply:
Just out of curiosity... why do you want them to be in hexadecimal? I like my bit masks in binary for example. Is there an advantage for hex specifically?

Reply:
Can be in binary also. Masks/numbers over 9 have different representation. Eg number 21
```
21 dec = 15 hex = 00010101 bin
21 hex = 21 hex = 00100001 bin
```
If we are talking about bitmasks we  usually care about binary format of number/about bits. And it is much easier to know what bits are 1 and 0 in hex or bin representation. There is no easy way to do it in decimal representation.
```
21 hex =  2 | 1 hex = 0010 | 0001  
```
It is enough just to divide number int characters and convert each one separately to 4bin numbers - easy to do it in your head.

Reply:
I agree. Bitmasks in hex are easier to read than decimal and shorter than binary. Imagine 32 bit binary bitmasks 😁

Reply:
Fair enough

Reply:
Agreed on using hex, and updated accordingly. As for right justification, not a good use of the space bar in my opinion.